### DISCUSSION TITLE
Ability to edit VN picture (when using vndb import) besides using .ico/.exe file

### DISCUSSION THREAD

**Discussion by superange128** [micah686/VnManager#25] [Ideas]
As far as I see the only way to edit an picture (havent actually tried yet) in VN Manager is to use a .ico/.exe file.

It would be nice to be able to use a different picture besides the default one vndb uses.

Maybe set some kind of resolution limitations so it can fit well.

**Comment by micah686** [comment]
The ico/exe is actually for the application's icon on the 'Main' tab. not for the actual picture. The ico/exe is typically going to be a roughly 32x32 image. And yes, for games marked as Vndb, there is currently no way to modify the cover image. I'll look into it, but I'll need to make sure my code will load the right image. The other issue about that is that I can't tell if a user supplied image is NSFW or not, so if the user adds their own image, my settings to block nsfw content would not work for those images.
